ui/src/app/components/breadcrumb.tsx

```diff
@@ -61,29 +61,41 @@ export const getTrail = (type: BreadcrumbType, data): {label: string, url: strin
       ];
     case BreadcrumbType.ConceptSet:
       return [
-        ...getTrail(BreadcrumbType.Workspace, data),
-        {label: conceptSet ? conceptSet.name : '...', url: `${prefix}/concepts/sets/${csid}`}
+        ...getTrail(BreadcrumbType.Data, data),
+        {label: conceptSet ? conceptSet.name : '...', url: `${prefix}/data/concepts/sets/${csid}`}
       ];
     case BreadcrumbType.Cohort:
       return [
-        ...getTrail(BreadcrumbType.Workspace, data),
-        {label: cohort ? cohort.name : '...', url: `${prefix}/cohorts/${cid}/review/participants`}
+        ...getTrail(BreadcrumbType.Data, data),
+        {label: cohort ? cohort.name : '...',
+          url: `${prefix}/data/cohorts/${cid}/review/participants`}
       ];
     case BreadcrumbType.Participant:
       return [
         ...getTrail(BreadcrumbType.Cohort, data),
-        {label: `Participant ${pid}`, url: `${prefix}/cohorts/${cid}/review/participants/${pid}`}
+        {label: `Participant ${pid}`,
+          url: `${prefix}/data/cohorts/${cid}/review/participants/${pid}`}
       ];
     case BreadcrumbType.CohortAdd:
       return [
-        ...getTrail(BreadcrumbType.Workspace, data),
-        {label: 'Build Cohort Criteria', url: `${prefix}/cohorts/build`}
+        ...getTrail(BreadcrumbType.Data, data),
+        {label: 'Build Cohort Criteria', url: `${prefix}/data/cohorts/build`}
+      ];
+    case BreadcrumbType.SearchConcepts:
```

Comment:
Note: I added this breadcrumb because it seemed weird that there wasn't one for Concept search.  will need a little more clean up with concepts/concept sets IMO, see my note in the description.